packages/webpack-scaffold/src/index.ts

```diff
@@ -109,7 +109,7 @@ export function InputValidate(
     message: string,
     cb?: (input: string) => string | boolean,
     defaultChoice?: string,
-    skip?: boolean,
+    skip = false,
```

Comment:
Why do you change it?

Reply:
Reverted.

Reply:
I would say, having it as a default parameter would be a better idea such that it doesn't stay `undefined`.

Reply:
Please avoid it, that PR is only tests, if you want to refactor it, do it in the other PR

Reply:
In that case, I guess we're good to land